filter/sentinel/filter.go

```diff
@@ -129,8 +126,9 @@ func (d *sentinelProviderFilter) Invoke(ctx context.Context, invoker protocol.In
 	if b != nil {
 		// interface blocked
 		return sentinelDubboProviderFallback(ctx, invoker, invocation, b)
+	} else {
+		defer interfaceEntry.Exit()
 	}
```

Comment:
`else` here could be removed.

```suggestion
	}
	defer interfaceEntry.Exit()
```